contracts/two-party-pol-holder/src/contract.rs

```diff
@@ -240,7 +241,10 @@ fn try_claim_share_based(
     covenant_config.update_parties(claim_party, counterparty);
 
     COVENANT_CONFIG.save(deps.storage, &covenant_config)?;
-    Ok(Response::default().add_messages(withdraw_and_forward_msgs))
+
+    Ok(Response::default()
+        .add_attribute("method", "claim_share_based")
+        .add_messages(messages))
```

Comment:
Just FYI, the add functions take `Into<CosmosMsg<T>>` so if you have a vector of the same type (WasmMsg) you just just pass it a vector of it and you don't have to wrap it in CosmosMsg type.
Just makes it easier on eye.

Reply:
Or, you can also do `WasmMsg {}.into()` if its a vector of `CosmosMsg` that way you don't need to have the long wrapper.

Reply:
Oh that's cool. It would affect pretty much every contract so I'll keep this clean and make an issue!